src/app/shared/components/Empty/Empty.tsx

```diff
@@ -22,8 +22,10 @@ export default function Empty({ icon, title, subtitle, action, contextMenuClick
       <button
         onClick={action.onClick}
         className={`mx-auto flex items-center rounded-lg ${
-          action.style === 'elevated' ? 'mt-5 bg-primary' : 'mt-2.5 bg-transparent'
-        } h-10 bg-opacity-10 px-5 font-medium text-primary hover:bg-opacity-15 active:bg-opacity-20`}
+          action.style === 'elevated'
+            ? 'mt-5 bg-primary/10 hover:bg-primary/15 active:bg-primary/20'
```

Comment:
`hover:bg-primary/15 active:bg-primary/20` should be removed from this line and added back to the line 28